src/App.js

```diff
@@ -1,26 +1,94 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import AddFood from './components/AddFood'
+import Foodbox from './components/FoodBox'
+
+class App extends Component {
+
+  state = {
+    foodList: foods,
+    filteredList: foods,
+    showForm: false
+  }
+
+
+
+  handleAdd = (event) => {
+    event.preventDefault()
+    
+
+    let food = event.target
+    let newFood = {
+        name: food.name.value,
+        calories: food.calories.value,
+        quantity: 0,
+        image: food.image.value, 
+    }
+
+    this.setState({
+      foodList: [...this.state.foodList, newFood],
+      filteredList: [...this.state.filteredList, newFood],
+      showForm: false
+    })
+
+    }
+
+
+
+    handleFormShow = () => {
+      this.setState({
+        showForm: true
+      })
+    }
+
+
+    handleSearch = (event) => {
+    event.preventDefault()
+    let search = event.target.value
+    const {foodList} = this.state
+
+    let searchedList = foodList.filter((food) => {
+      return food.name.toLowerCase().includes(search.toLowerCase())
+    })
+
+    this.setState({
+      filteredList: searchedList
+    })
+    }
```

Comment:
These two closing brackets should not be on the same indentation level. Remember to pay attention at code cleanness to avoid this issues or use the auto format of VS code.